<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClass</class>
 <widget class="QWidget" name="WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="msgBrowser"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QFontComboBox" name="fontCbx"/>
         </item>
         <item>
          <widget class="QComboBox" name="sizeCbx">
           <property name="currentIndex">
            <number>4</number>
           </property>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>17</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="boldTBtn">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>加粗</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="italicTBtn">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>倾斜</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="underlineTBtn">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>下划线</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="colorTBtn">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>更改字体颜色</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="sendTBtn">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>传输文件</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="saveTBtn">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>保存聊天记录</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="clearTBtn">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>清空聊天记录</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="msgTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="usrTblWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>用户名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>IP</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sendBtn">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="usrNumLbl">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Widget.qrc"/>
 </resources>
 <connections/>
</ui>
